pwiz_tools/Skyline/Executables/Installer/Product-template.wxs

```diff
@@ -114,12 +114,20 @@
         <RemoveFolder Id="ApplicationProgramsFolder" On="uninstall"/>
         <RegistryValue Root="HKCU" Key="Software\MacCossLabUW\Skyline64" Name="installed" Type="integer" Value="1" KeyPath="yes"/>
       </Component>
+      <Component Id="ApplicationBatchShortcut64">
+        <Shortcut Id="ApplicationBatchMenuShortcut32" Name="$(var.ProductName) Batch" Target="[#SkylineBatch_exe]"/>

```

Comment:
"$(var.ProductName) Batch" ends up appearing on the Start Menu as:
Skyline-Daily (64 bit) Batch
I think we should change this to:
"$(var.Skyline.TargetName) Batch (64)"
which I think would end up looking like:
Skyline-Daily Batch (64 bit)